src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -38,7 +38,7 @@ trait RawlsBillingProjectComponent {
     def save(billingProject: RawlsBillingProject): WriteAction[RawlsBillingProject] = {
       val projectInsert = rawlsBillingProjectQuery insertOrUpdate marshalBillingProject(billingProject)
       val userInserts = billingProject.users.toSeq.map {
-        projectUsersQuery insertOrUpdate marshalProjectUsers(_, billingProject)
+        projectUsersQuery insertOrUpdate marshalProjectUsers(_, billingProject.projectName)
```

Comment:
code elsewhere that calls marshalProjectUsers needed to change so that further up the chain we do not have to do the extra query to load the project